### PR TITLE
[full-ci] Move OcStatusIndicators in separate column

### LINKED ISSUE #5976: Move share indicators into their own column
## Acceptance criteria
- load share indicators for all file listing views in an authenticated context (`All files`, `Favorites`, `Shared with others`, `Shared via link`, if possible also for `Trashbin`)
- add share indicators into a dedicated column (directly after resource name) with title `Shared`
- accessible names for share indicators `Shared via link` for share type link, `Shared with people` for all other share types
- make share indicators prettier (@tbsbdr for scribbles)
- also make sure that the same share indicator look and feel is used in the details panel
- remove share indicators from below the resource name
- remove avatars list as dedicated column in the `Shared with others` and "shared via link" views. "shared with me" stays untouched. That costs us too much performance, especially rendering performance and the UX is confusing (they are not clickable, it is not useful to make them clickable because the click behaviour is not clear and cannot be directed into the dedicated panels, users expect to be able to click)
- click on a share indicator opens the respective right sidebar panel (people or links)

### PR DESCRIPTION
**Pull request by lookacat**
## Related Issue
- Fixes #5976

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/src/components/FilesList/ResourceTable.vue [hunk 1]
```diff
@@ -394,6 +401,8 @@ export default {
         })
       }
       const sortFields = determineSortFields(firstResource)
+      console.log('FIRST RESOURCE')
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/FilesList/ResourceTable.vue:404]: Dev leftover

File: packages/web-app-files/src/components/FilesList/ResourceTable.vue [hunk 2]
```diff
@@ -403,6 +412,13 @@ export default {
             width: 'expand',
             wrap: 'truncate'
           },
+          {
+            name: 'shareTypes',
+            title: this.$gettext('Shares'), // How do we get the translations here?
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/FilesList/ResourceTable.vue:417]: `this.$gettext(` should work fine here

File: changelog/unreleased/enhancement-move-share-indicators [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Enhancement: Move share indicators
+
+We've moved the share/status indicators into a seperate column
+and adjusted the design in ODS.
+
+https://github.com/owncloud/web/issues/5976
+https://github.com/owncloud/owncloud-design-system/pull/2014
```
Comment (pascalwengerter) [inline changelog/unreleased/enhancement-move-share-indicators:7]:
```suggestion
https://github.com/owncloud/owncloud-design-system/pull/2014
https://github.com/owncloud/web/pull/6552
```

**Comment by ownclouders** [comment]
Results for <strong>oCISTrashbinUploadMoveJourney</strong> https://drone.owncloud.com/owncloud/web/23453/69/1
The following scenarios passed on retry:
- webUITrashbinDelete/trashbinDelete.feature:56

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingPermToRoot</strong> https://drone.owncloud.com/owncloud/web/23528/35/1
The following scenarios passed on retry:
- webUISharingPermissionToRoot/sharePermissionsUsers.feature:112

**Comment by pascalwengerter** [comment]
@lookacat I marked this as ready for review since that's the case, from my side. Last commit should exclude failing tests on tablet resolution, `webUITrashbinRestore/trashbinRestore.feature:14` is just pretty flaky (at least locally had a ~70% success chance). I'd ask @kulmann for a review since I contributed here myself

**Review by kulmann — APPROVED** [review]
LGTM 👍

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/23541/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23541/tracing/alice-shares-folder-with-file-to-brian-alice-2022-3-10-05-13-43.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp1</strong> https://drone.owncloud.com/owncloud/web/23544/71/1
The following scenarios passed on retry:
- webUIPrivateLinks/accessingPrivateLinks.feature:9

**Pull request merged**